tests/jsx_escape/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -42,9 +42,9 @@ raw_amp = <span>foo & bar</span>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 many_nbsp = <div>&nbsp; &nbsp; </div>;
 single_nbsp = <div>&nbsp;</div>;
-many_raw_nbsp = <div>&nbsp;&nbsp;&nbsp;</div>;
+many_raw_nbsp = <div>   </div>;
```

Comment:
This is still not correct. `&nbsp;` is different than a normal space. They need to be preserved.